language-design.md

```diff
@@ -59,50 +72,45 @@ What does the post on grayscale tell us about the process of API design?
 [[Cook 2007, page 1-20]](https://dl.acm.org/citation.cfm?doid=1238844.1238845)
 
-Are these two experiences of natural languages at odds with one another? Would
-you choose to include natural language in the design of a DSL? If so, how might
-you do so? If not, why not?
+Are these two experiences of natural languages at odds with one another? Would you choose to include natural language in the design of a DSL? If so, how might you do so? If not, why not?
 
 **Response**
 
+There are two ways to look at this dichotomy between programming languages which have the feel of a "natural language" being both better and worse for new programmers.
 
+We may note that there is a spectrum to how closely the language resembles natural language. On one end, we have the binary / hexidecimal codes in which computers store their instructions, and on the other side we have languages, like AppleScript, which *read* much like a natural language. One might note that both of these extremes are poor choices for programmers. The former, while entirely precise and unambiguous is not structured, nor is it easy to see patterns or errors.  The latter on the other hand depends on the natural languages, which are full of their own ambiguities and exceptional grammars. Thus, one might argue that the correct programming language is somewhere in between these two extrema, and the language tested in the Palvus article may have been at a good location along this spectrum, while a language like AppleScript was too close to the ambiguous, hard-to-tell-what's-a-keyword, syntax of natural languages.
+
+On the other hand, we may note that rather than a simple one dimensional spectrum of "how close to natural language is this?", the usability of the language may actually depend on _how_ the natural language is employed. For example, in AppleScript, it seemed that the natural language took the form of actually creating sentences. There would be examples such as `tell application X to quit`, which includes the optional (but not _obviously_ optional) command: `to`. The natural language syntax in the Palvus article seemed to rather import words from natural language whose definition was already quite clear and allow their judicious use to make commands and control flow clearer to a new programmer -- compared to the curly brace and symbols structure which is more concise but described as obtuse by the Palvus article.
+
+So it might be wise to include parts of natural languages that are concise, recognizable-as-clear-patterns and intuitive to read *and* write; To specify, an example of a natural-language word that is intuitive to write and read would be `end`, and that of a word that is intuitive to read but not to write would be `repeat`. 
```

Comment:
I think you have a similar thirst for balance that we had; however, I think that an "intuitive" language to write and read might be difficult. For example, I usually think when writing that it would be intuitive to say `if MyGrades are greater than 'A' ..` instead of `if MyGrades is greater than 'A'..` because MyGrades is "plural" but in a language these little differences can make writing and reading confusing. 

Maybe intuitiveness has to be a part of the balance?